dashboard/lti_new.py

```diff
@@ -234,6 +235,18 @@ def extract_launch_variables_for_tool_use(request, message_launch):
         password = ''.join(random.sample(string.ascii_letters, settings.RANDOM_PASSWORD_DEFAULT_LENGTH))
         user_obj = User.objects.create_user(username=username, email=email, password=password, first_name=first_name,
                                             last_name=last_name)
+
+    # Add username into the MyLA User table, since the data was not pulled in from cron job
+    user_id = CANVAS_DATA_ID_INCREMENT + int(canvas_user_id)
+    try:
+        myla_user_obj = MyLAUser.objects.get(user_id=user_id)
+        # update
+        myla_user_obj.name = full_name
```

Comment:
I did a quick search, I don't think we're using `MyLAUser.full_name`. Thoughts on dropping this field now?

Reply:
Dropped the field.